src/NuGetGallery.Core/GitHub/NuGetPackageGitHubInformation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class NuGetPackageGitHubInformation
+    {
+        public int TotalRepos { get; set; }
+        public IReadOnlyList<RepositoryInformation> Repos { get; set; }
+
+        public NuGetPackageGitHubInformation()
```

Comment:
I would make the constructor accept `totalRepos` and `repos` parameters, and then remove the `set;` from the properties.